app/gui/view/graph-editor/src/builtin/visualization/native/inc/error_preprocessor.enso

```diff
@@ -2,6 +2,5 @@ x ->
     result = Ref.new '{ message: ""}'
```

Comment:
Does this file not need the import statement now?

Reply:
it's a bit more tricky and the test would be failing on this one (hence this being a draft still). I've been talking to @4e6 about it because it is not possible to add the import here and `error_preprocessor` executes in the context of `Standard.Base.Main`. We will likely rewrite this piece of code to not use `Ref`